### PR TITLE
[MOB-3833] No coverage screen

### PR DESCRIPTION
**Pull request by EdwardW2**
## JIRA
[MOB-3833](https://karhoo.atlassian.net/browse/MOB-3833)

## DESCRIPTION
[Create a description of what you have changed in this branch]

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [ ] New feature 
- [ ] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [x] Other: *define what type of change it is*

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [x] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [x] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

File: uisdk/src/main/java/com/karhoo/uisdk/screen/booking/quotes/errorview/QuotesErrorView.kt [hunk 1]
```diff
@@ -46,9 +47,10 @@ class QuotesErrorView @JvmOverloads constructor(
         quotesErrorSubtitle.text = presenter.setSpannableOnSubititle(
             reason.linkTitle,
             reason.subtitleLink,
-            reason.title,
+            reason.subtitle,
```
Comment (mateid-kr) [inline uisdk/src/main/java/com/karhoo/uisdk/screen/booking/quotes/errorview/QuotesErrorView.kt:50]: Good catch!

Comment (mateid-kr) [inline uisdk/src/main/java/com/karhoo/uisdk/screen/booking/quotes/errorview/QuotesErrorView.kt:50]: Good catch!

File: uisdk/src/main/java/com/karhoo/uisdk/screen/booking/quotes/list/QuotesRecyclerView.kt [hunk 2]
```diff
@@ -126,18 +128,24 @@ class QuotesRecyclerView @JvmOverloads constructor(
...
             quotesErrorView.visibility = View.VISIBLE
             quotesErrorView.setup(
-                ErrorViewGenericReason(
-                    context.resources.getString(R.string.kh_uisdk_no_availability_title),
-                    context.resources.getString(R.string.kh_uisdk_no_availability_subtitle),
-                    R.drawable.kh_uisdk_ic_no_available_quotes
+                ErrorViewLinkedReason(
+                    context.resources.getString(R.string.kh_uisdk_no_coverage_title),
+                    context.resources.getString(R.string.kh_uisdk_no_coverage_subtitle),
+                        "Contact us",
```
Comment (mateid-kr) [inline uisdk/src/main/java/com/karhoo/uisdk/screen/booking/quotes/list/QuotesRecyclerView.kt:134]: This should be localized

**Review by mateid-kr — APPROVED** [review]

**Review by mateid-kr — CHANGES_REQUESTED** [review]

**Review by dianakarh — COMMENTED** [review]
CircleCI failing because of missing translations:
Errors found:
  
  /home/circleci/project/uisdk/src/main/res/values/strings.xml:274: Error: "kh_uisdk_no_availability_subtitle" is not translated in "de" (German), "en" (English), "it" (Italian), "fr" (French), "es" (Spanish), "nl" (Dutch) [MissingTranslation]
      <string name="kh_uisdk_no_availability_subtitle">Please select another time</string>
              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  /home/circleci/project/uisdk/src/main/res/values/strings.xml:275: Error: "kh_uisdk_no_availability_title" is not translated in "de" (German), "en" (English), "it" (Italian), "fr" (French), "es" (Spanish), "nl" (Dutch) [MissingTranslation]
      <string name="kh_uisdk_no_availability_title">No availability for the requested time</string>

**Review by mateid-kr — APPROVED** [review]

**Pull request merged**